blockchyp/blockchyp_client.py

```diff
@@ -3,20 +3,23 @@
 """
 
 
+from datetime import datetime
 import requests
+from .blockchyp_crypto import CryptoUtils
 
 
 class BlockChypClient:
     """
     Provides integration with the BlockChyp gateway and payment terminals for python developers
     """
-    def __init__(self, creds):
-        self.gateway_host = "https://api.blockchyp.com"
+    def __init__(self, creds, https=False):
+        # self.gateway_host = "https://api.blockchyp.com" # live server
+        self.gateway_host = "https://test.dev.blockchyp.com" # test server
```

Comment:
you should probably make test gateway and production gateway configuration options
recommending adding self.test_gateway_host and toggling between them when assembling urls based on the test flag.